luminance-windowing/src/lib.rs

```diff
@@ -78,6 +82,19 @@ impl WindowOpt {
   pub fn is_cursor_hidden(&self) -> bool {
     self.hide_cursor
   }
+
+  #[inline]
+  pub fn set_num_samples(self, samples: Option<u32>) -> Self {
```

Comment:
Ditto.

Reply:
Also, we can use `samples: S where S: Into<Option<u32>>`, I prefer that kind of interface.